### PR TITLE
Remove @types/lodash.sortBy from project

### LINKED ISSUE #1930: Remove unused `@types/lodash.sortby` package
## Problem

`@types/lodash.sortby` is installed in the frontend/package.json but `lodash.sortBy` is not used in our project, nor are these types relied on otherwise.

## Description

Remove the `@types/lodash.sortby` entry from package.json and update the lock file.

### PR DESCRIPTION
**Pull request by wasimTQ**
## Fixes

Fixes #1930 by @sarayourfriend 

## Description

This PR removes `@types/lodash.sortBy`. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sarayourfriend** [comment]
Thanks for the contribution, @wasimTQ, and welcome to Openverse! To complete this PR, you'll need to update `pnpm-lock.yaml` as well. To do so, run `pnpm install` locally in this branch and commit the changes to `pnpm-lock.yaml`.

**Comment by wasimTQ** [comment]
> Thanks for the contribution, @wasimTQ, and welcome to Openverse! To complete this PR, you'll need to update `pnpm-lock.yaml` as well. To do so, run `pnpm install` locally in this branch and commit the changes to `pnpm-lock.yaml`.

Got it. I was wondering why it's failing

**Comment by wasimTQ** [comment]
@obulat @sarayourfriend When I ran the precommit locally, it worked. I don't know why it's failing on PR.

Should I remove the type and use it as default export as it mentioned?

File: pnpm-lock.yaml
Comment (obulat) [inline pnpm-lock.yaml:1]: The lock file appears to have changes for a lot of unrelated packages, which causes failures in the linting.
When I removed `@types/lodash.sortby` locally, and ran `just node-install, I only saw 4 changes in `pnpm-lock.yaml` file. Could you check the version of `pnpm` you are using? It should match the version in `package.json`'s engines property to generate the correct lock file.

**Review by obulat — CHANGES_REQUESTED** [review]
The lock file should not contain so many changes. The pnpm version mismatch might have caused this, but I'm not entirely sure.

File: pnpm-lock.yaml
Comment (wasimTQ) [reply pnpm-lock.yaml:1]: But it passes on local. Let me try after correcting the version

Comment (wasimTQ) [reply pnpm-lock.yaml:1]: Can you confirm it once the pipeline finish?

**Review by obulat — APPROVED** [review]

**Comment by obulat** [comment]
All the checks are 🟢 now! Thank you for your contribution, @wasimTQ 🎉

**Review by sarayourfriend — APPROVED** [review]

**Pull request merged**